static/nearby_map.html

```diff
@@ -14,6 +14,7 @@
     -->
     <link rel="stylesheet" href="css/main.css"/>
     <link rel="stylesheet" href="resources/fontawesome-free-5.13.0-web/css/all.min.css">
+    <script crossorigin="anonymous" src="https://polyfill.io/v3/polyfill.min.js"></script>
```

Comment:
The good thing about this is that in newer browsers the bundle is small or empty